Libraries/Text/RCTTextField.m

```diff
@@ -112,6 +113,85 @@ - (void)setPlaceholder:(NSString *)placeholder
   RCTUpdatePlaceholder(self);
 }
 
+- (UIColor *)tintColor
+{
+  return super.tintColor;
+}
+
+- (void)setTintColor:(UIColor *)tintColor
+{
+  super.tintColor = tintColor;
+}
+
+- (UIColor *)clearButtonTintColor
+{
+  return _clearButtonTintColor;
+}
+
+- (void)setClearButtonTintColor:(UIColor *)clearButtonTintColor
+{
+    _clearButtonTintColor = clearButtonTintColor;
+}
+
+static UIButton* findClearButton(RCTTextField *self)
+{
+    for(UIView *v in self.subviews)
+    {
```

Comment:
Curly braces on the same line for loops, if-clauses, etc. Look at other places in the file to infer the coding conventions.